api-tests/server/checks_test.go

```diff
@@ -277,17 +270,8 @@ func TestChangeSecurityChecks(t *testing.T) {
 			assert.Equal(t, "RARE", *check.Interval)
```

Comment:
We should only compare checks that we modified. New checks can be added between our update and list API calls.

Reply:
We don't run these tests in parallel and we collect check files only once in that test group. So It seems that the check set will be always the same inside this test. Correct me if I'm wrong.